app/models/assignment.rb

```diff
@@ -625,7 +615,7 @@ def summary_csv(user)
     end
 
     headers = [['User name', 'Group', 'Final grade'], ['', 'Out of', self.max_mark]]
-    criteria = self.get_criteria(:ta)
+    criteria = self.ta_criteria
```

Comment:
If the only usage of this variable is on the previous line, remove this variable and just use `self.ta_criteria` on the line below.